docs/pages/next-steps/additional-resources.md

```diff
@@ -2,45 +2,39 @@
 title: Additional resources
 ---
 
-The following resources are useful for learning Expo and some of the projects that it depends on.
+The following resources are useful for learning about Expo tooling and services.
 
 ## Expo Blog
 
 - [Exposition](https://blog.expo.dev/) - our official blog, where we post release notes every month and other Expo related content at random intervals.
 
-## Courses using Expo
+## GitHub
 
-- [Stanford CS 47SI Cross-Platform Mobile Development](https://web.stanford.edu/class/cs47si/) (free)
-- [University of Helsinki - Full Stack Open](https://fullstackopen.com/en/part10) (free)
-- [React Europe - Introduction to React Native Workshop videos on YouTube](https://www.youtube.com/playlist?list=PLCC436JpVnK2RFms3NG9ubPToWCNbMLbT) (free)
-- [Udacity - React Nanodegree](https://www.udacity.com/course/react-nanodegree--nd019) (paid)
+- [expo/expo](https://github.com/expo/expo): Expo Go, SDK, Docs, and the Expo CLI.
+- [expo/eas-cli](https://github.com/expo/eas-cli): Fastest way to build, submit, and update iOS and Android apps.
+- [expo/examples](https://github.com/expo/examples): Integrations and other examples.
+- [expo/config-plugins](https://github.com/expo/config-plugins): Expo Config Plugins for working with third-party packages.
+- [expo/snack](https://github.com/expo/snack): Build apps from the browser.
+- [expo/vscode-expo](https://github.com/expo/vscode-expo): VS Code extension for working with Expo tools.
+- [expo/fyi](https://github.com/expo/fyi): Troubleshooting guides for Expo tools and services.
```

Comment:
```suggestion
- [expo/expo](https://github.com/expo/expo) - Expo Go, SDK, Docs, and the Expo CLI.
- [expo/eas-cli](https://github.com/expo/eas-cli) - Fastest way to build, submit, and update iOS and Android apps.
- [expo/examples](https://github.com/expo/examples) - Integrations and other examples.
- [expo/config-plugins](https://github.com/expo/config-plugins) - Expo Config Plugins for working with third-party packages.
- [expo/snack](https://github.com/expo/snack) - Build apps from the browser.
- [expo/vscode-expo](https://github.com/expo/vscode-expo) - VS Code extension for working with Expo tools.
- [expo/fyi](https://github.com/expo/fyi) - Troubleshooting guides for Expo tools and services.
```

Suggesting to use `-` instead of `:` to be consistent and match the style of the rest of the document.